docs/RFPs/polkadot-js-apps.md

```diff
@@ -0,0 +1,131 @@
+
+### Maintenance list
+
+The following is a list of repos that need to be maintained and potentially further developed:
+
+- https://github.com/polkadot-js/apps
+- https://github.com/polkadot-js/docs
+- https://github.com/polkadot-js/tools
+- https://github.com/polkadot-js/api
+- https://github.com/polkadot-js/extension
+- https://github.com/polkadot-js/common
+- https://github.com/polkadot-js/dev
+- https://github.com/polkadot-js/ui
+- https://github.com/polkadot-js/wasm
+- https://github.com/polkadot-js/build-deno.land
+- https://github.com/polkadot-js/build-bundle
+- https://github.com/polkadot-js/phishing
+
+## Maintenance Responsibilities :nut_and_bolt:
+
+This section should specify what kind of tasks and responsibilities your maintainer team will cover during the contract period. Feel free to outline a list of issues/bugs or pull requests that need further development - you can specify them here to provide more context on what tasks you will close. You are also free to argue whether it would make more sense to transfer the repos to your organization, or to work on a fork.
+
+Some example responsibilities might be:
+
+- We will maintain the project's documentation.
+- We will answer issues and discussions.
+- We will look to resolve issues and merge open pull requests.
+- We will fix new bugs as we receive and help contributors who have pull requests get any needed information.
+
+> ⚠️ *Note that all code should be published under an open-source license during the maintenance period.*
+
+## Technical Considerations
+
+- **Roadmap:** What is the team's roadmap for future development and updates? How will they go about prioritizion?
+- **Strategy:** How will the team manage bug fixes, new features, and community contributions?
+- **Testing and CI/CD:** Will the team keep the current infrastructure or replace it?
+- **Supported Frameworks and Libraries:** Do you foresee any potential maintenance challenges? Is there anything that requires special attention?
+
+
+## Deliverables :nut_and_bolt:
+
+The following items could be the initial deliverables of the project. These are just examples of course; improvements/additions are welcome. The listed costs are placeholders.
+
+### Overview
+
+- **Start Date:** Date, when you plan to start with the maintenance work.
+- **Sprint/Period Duration:** Duration of the sprint/period (e.g. 4 weeks)
+- **Total Duration:** Duration of the entire maintenance contract (e.g. 2 years)
+- **Full-Time Equivalent (FTE):**  Average number of full-time employees working on the project throughout its duration (see [Wikipedia](https://en.wikipedia.org/wiki/Full-time_equivalent), e.g. .5 FTE)
+- **Max budget per sprint/period:** Requested max budget in USD per sprint/period (e.g. 7,000 USD).
+- **Hourly rate:** Amount of budget per hour, since it’s unlikely that the maintenance of the project requires the exact same workload each sprint.
+
+### Milestone 1 - Initial Stabilization
```

Comment:
We don't usually ask for milestones for a maintenance grant. It's at odds with the "max budget per sprint" info above. Perhaps we can ask for a "thorough review" for the first sprint and lists of issues for the first few sprints, instead of milestones?